### PR TITLE
[fix] Split sentences for translation (#828)

### PR DESCRIPTION
**Pull request by opi**
Work in progress. Fixes https://dev.yunohost.org/issues/828

Original string should be deleted once new translations have been done.

There is probably more string to split if we wal through that path. I let @Jibec point them out ;)

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -63,14 +63,24 @@
...
     "confirm_delete": "Are you sure you want to delete %s ?",
     "confirm_firewall": "Are you sure to %s port %s (protocol: %s, connection: %s)",
+    "confirm_firewall_open": "Are you sure to open port %s (protocol: %s, connection: %s)",
+    "confirm_firewall_close": "Are you sure to close port %s (protocol: %s, connection: %s)",
     "confirm_install_custom_app": "Installing 3rd party applications may compromise the security of your system. Use at your own risk.",
     "confirm_install_domain_root": "You will not be able to install any other app on %s. Continue ?",
     "confirm_postinstall": "You are about to launch the post-installation process on the domain %s. It may take a few minutes, *do not interrupt the operation*.",
     "confirm_restore": "Are you sure you want to restore %s ?",
     "confirm_service_action": "Are you sure you want to %s %s ?",
+    "confirm_service_start": "Are you sure you want to start %s ?",
```
Comment (M5oul) [inline src/locales/en.json]: [typo] No space before `?`.
Same for following added lines.

Comment (opi) [inline src/locales/en.json]: Actually there are looots of "spaces before question mark" in `en.json`, and it's probably out of scope to catch them all now. I have to try if `%s ?` won't break our translation system though.

Comment (M5oul) [inline src/locales/en.json]: Don't modify other strings. Just the one you added.

**Comment by opi** [comment]
( remind to squash before merge )

**Comment by Jibec** [comment]
Le 21 mars 2017 16:38:16 UTC+01:00, opi <notifications@github.com> a écrit :
>Work in progress. Fixes https://dev.yunohost.org/issues/828
>
>Original string should be deleted once new translations have been done.
>
>There is probably more string to split if we wal through that path. I
>let @Jibec point them out ;)
>You can view, comment on, or merge this pull request online at:
>
>  https://github.com/YunoHost/yunohost-admin/pull/153
>
>-- Commit Summary --
>
>  * [fix] Split confirm_update sentence for better translations.
> * [fix] Split confirm_service_action sentence for better translations.
>  * [fix] Split confirm_firewall sentence for better translations.
>  * [fix] Split confirm_upnp_action sentence for better translations.
>
>-- File Changes --
>
>    M src/js/yunohost/controllers/firewall.js (6)
>    M src/js/yunohost/controllers/services.js (2)
>    M src/js/yunohost/controllers/tools.js (2)
>    M src/locales/en.json (10)
>
>-- Patch Links --
>
>https://github.com/YunoHost/yunohost-admin/pull/153.patch
>https://github.com/YunoHost/yunohost-admin/pull/153.diff

Thanks!
You should delete old strings, the translation platform has a translation memory. Even if it doesn't, it isn't much more work.

Some typo to fix:
confirm_update_apps, confirm_update_packages: missing -to-

confirm_firewall_open/close: missing -you want-
-- 
Jean-Baptiste Holcroft

**Comment by opi** [comment]
@Jibec I fixed the tyops you pointed out. Could you please approve this pull request if it looks good to you ?

**Comment by Jibec** [comment]
ok pour moi, merci beaucoup

**Comment by M5oul** [comment]
You forget to squash.

**Comment by opi** [comment]
@M5oul nope https://github.com/YunoHost/yunohost-admin/commit/dc84ce7224b7a54ef4909e7792b21a2c1bc677d1

**Pull request merged**